diff --git a/tensorflow/core/platform/platform.h b/tensorflow/core/platform/platform.h
index 0481b36..7a07628 100644
--- a/tensorflow/core/platform/platform.h
+++ b/tensorflow/core/platform/platform.h
@@ -40,13 +40,13 @@ limitations under the License.
 #elif defined(_WIN32)
 #define PLATFORM_WINDOWS
 
-#elif defined(__arm__)
+#elif defined(__arm__) || defined(__aarch64__)
 #define PLATFORM_POSIX
 
 // Require an outside macro to tell us if we're building for Raspberry Pi or
 // another ARM device that's not a mobile platform.
 #if !defined(RASPBERRY_PI) && !defined(ARM_NON_MOBILE)
-#define IS_MOBILE_PLATFORM
+//#define IS_MOBILE_PLATFORM
 #endif  // !defined(RASPBERRY_PI) && !defined(ARM_NON_MOBILE)
 
 #else
